etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Thanks. That is required sooner as other PRs need to update based on that reference one.